src/components/timeline/RecentTimeline.js

```diff
@@ -1,25 +1,63 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import axios from 'axios';
-import { useQuery } from 'react-query';
 import { Timeline, TimelineItem } from 'vertical-timeline-component-for-react';
-import Moment from 'react-moment';
+import { DateTime } from 'luxon';
+import { nanoid } from 'nanoid';
+import { useTimeline } from '../../state/query_hooks/useTimeline';
+import './RecentTimeline.css';
 
 export function RecentTimeline() {
-  useQuery('timeline', async () => {
-    const { data } = await axios.get(
-      `${process.env.REACT_APP_BACKENDURL}/incidents/getincidents`
-    );
+  const timelineQuery = useTimeline();
 
-    return data;
-  });
+  const urlDomain = url => {
+    let re = /^(?:https?:\/\/)?(?:[^@\n]+@)?(?:www\.)?([^:/\n?]+)/gim;
+    let newUrl = url.split(re)[1].replace('.com', '');
+    return newUrl;
+  };
 
-  return (
-    <div>
-      Timeline
-      <ul></ul>
+  return timelineQuery.isSuccess ? (
+    <div className="timeline-container">
+      <h1> Timeline of Recent Events </h1>
+      <Timeline lineColor={'#</Timeline>ddd'}>
+        {timelineQuery.data.map(details => {
+          return (
+            <TimelineItem
```

Comment:
I think we should consider moving this TimeLineItem component to its own file. It has a few moving pieces and I think digesting it on its own will be simpler for an unfamiliar dev. It also increases the maintainability of the component because now someone can work on changes to the list item while someone does other changes to the list itself.

Non-critical

Reply:
So we can save this for the second release? So the PR can be merged after the testing is finished?